Rollbar/RollbarConfig.cs

```diff
@@ -0,0 +1,31 @@
+namespace RollbarDotNet
+{
+    public class RollbarConfig
+    {
+        public RollbarConfig() : this("")
+        {
+        }
+
+        public RollbarConfig(string accessToken)
+        {
+            AccessToken = accessToken;
```

Comment:
Is it convention in .NET to provide a constructor like this and then for the user of the library to call all of the setters to modify the default values? Or, is it common to add in different constructors with the different default values?